themes/ssp/attrauthcomanage/noty.tpl.php

```diff
@@ -77,7 +77,9 @@
     <?php if (!empty($this->data['noty']['icon'])) : ?>
     <img class="gif-box" src="resources/icons/<?= $this->data['noty']['icon'] ?>">
     <?php endif; ?>
-    <p><?= $this->t('{themeegi:attrauthcomanage_noty:title}') ?></p>
+    <?php if(!empty($this->data['noty']['title'])): ?>
```

Comment:
Replace line with `<?php if (!empty($this->data['noty']['title'])) : ?>`

Reply:
Done